### ISSUE TITLE
csminit shape defaults differ from spiceinit

### ISSUE THREAD

**Issue by jlaura** [DOI-USGS/ISIS3#5385]
**ISIS version(s) affected**: 8.0.0

**Description**  

`csminit` defaults to `none` for the shape model resulting in a bi-axial ellipsoid being used as the underlying shape model. In isolation, that is absolutely fine. `spiceinit` defaults to searching the system for a DEM to be used (in the base area). This can results in vastly different results when running `*init` -> `cam2map`. 

**How to reproduce**  

Run a cube through spiceinit -> cam2map and then run the same cube through ale -> csminit -> cam2map in a region with some topographic relief (for example, the ~800m of relief at the large lunar crater (-1˚, 109˚) is plenty to see large offsets).

**Possible Solution**  

The default for csminited cubes in ISIS should match the spiceinit defaults so that user expectations can be maintained across applications. Right now, a user has to read and grok the spiceinit and csminit docs, know that offsets can be a function of the underlying shape model, and then discover that the defaults are different. In sum, that is a non-trivial lift for a user.

**Additional context**

**Comment by acpaquette** [comment]
Something to note is that the USGSCSM doesn't have a triaxial radii solve for `imageToGround` it only deals with the math for a biaxial sphereoid. In other words. I don't think this is a bug but a feature of the library. We may want to transition this to a feature request rather than a bug fix.

**Comment by jlaura** [comment]
@acpaquette I did not mention triaxial support anywhere in the report. The issue is that csminit defaults to an ellipsoid while sppiceinit defaults to a system supplied shape model. The differences when projecting / orthorectifying are large and poorly documented.

**Comment by acpaquette** [comment]
Right, my bad. I forgot that the CSM Camera doesn't do an `imageToGround` call but gets the position and look vector and then does a target intersection through ISIS

**Comment by acpaquette** [comment]
Something to note here is that the CSMCamera uses the semi major and semi minor radii defined in the CSM model when defining the ISIS target. In some cases this won't matter as NAIF defines the target as a bi-axial ellipsoid, but it will matter if the target is defined as a tri-axial ellipsoid.

It would make sense here to have the CSMCamere follow what other ISIS cameras do. So if a target has been defined in ISIS as a tri-axial ellipsoid we should use that over the bi-axial defined in the CSM model

**Comment by Kelvinrr** [comment]
bump
